lib/helpers.js

```diff
@@ -125,6 +126,16 @@ export const createProfile = async ({
     // set `useCache: false` to fetch a fresh copy
     profileManager.getProfile({id: profileId, useCache: false})
       .catch(() => {});
+    const {name, color, shared} = profileContent;
+    if(name) {
+      profile.name = name;
+    }
+    if(color) {
+      profile.color = color;
+    }
+    if(shared) {
+      profile.shared = shared;
+    }
```

Comment:
Hmm, I think we should generalize this. If it's always the case that there are fields that are in `profileContent` that are added directly to the profile like this ... then we should just add them all in with, e.g. a loop, regardless of what the fields are. IOW, don't call out three specific fields, just add everything from `profileContent` into `profile`. We probably have to make an exception for `id` though -- don't override that.

Reply:
It looks like something similar to this will need to be done:

https://github.com/digitalbazaar/bedrock-web-wallet/blob/472835a0d6ba29c81f6f2474989bee4262b0c7c1/lib/helpers.js#L205-L219

Reply:
Probably the best thing to do is to create a local helper function called like `_mergeProfileContent({profile, profileContent} = {})` ... and have it get called for both the existing case above and this new case. It can return the merged content. So the existing code would change to `updatedDoc.content = _mergeProfileContent({profile: updatedDoc.content, profileContent})` and the new usage would be: profile = _mergeProfileContent({profile, profileContent})`.